src/azure-cli/azure/cli/command_modules/acr/check_health.py

```diff
@@ -123,16 +116,16 @@ def _get_docker_status_and_version(ignore_errors, yes):
 
 # Get current CLI version
 def _get_cli_version():
-    acr_component_name = "azure-cli-acr"
-    acr_cli_version = "not found"
     from pkg_resources import working_set
-    for component in list(working_set):
-        if component.key == acr_component_name:
-            acr_cli_version = component.version
 
-    print('ACR CLI version: {}'.format(acr_cli_version))
+    # working_set.by_key is a dictionary with component names as key
+    cli_component_name = "azure-cli"
+    cli_version = "not found"
+
+    if cli_component_name in working_set.by_key:
+        cli_version = working_set.by_key[cli_component_name].version
 
-    return 0
```

Comment:
This method just prints the CLI version. Calling function doesn't need any return value. Hence, removed.